unit-test/js-unit-testing_people/index.test.js

```diff
@@ -48,22 +48,74 @@ test("getFirstNames() returns an array with the first names", () => {
   expect(result).toEqual(["Jane", "John", "Max", "Erika", "Luke"]);
 });
 
-test('getFullNames() returns an array with full names as strings, e.g. "Jane Doe"', () => {});
+test('getFullNames() returns an array with full names as strings, e.g. "Jane Doe"', () => {
+  const result = getFullNames(people);
+  expect(result).toEqual([
+    "Jane Doe",
+    "John Doe",
+    "Max Mustermann",
+    "Erika Musterfrau",
+    "Luke Skywalker",
+  ]);
+});
 
-test('getNameAndAge() returns an array of formatted strings like this: "Skywalker (23)"', () => {});
+test('getNameAndAge() returns an array of formatted strings like this: "Skywalker (23)"', () => {
+  const result = getNameAndAge(people);
+  expect(result).toEqual([
+    "Doe (21)",
+    "Doe (18)",
+    "Mustermann (32)",
+    "Musterfrau (38)",
+    "Skywalker (23)",
+  ]);
+});
 
-test("getPeopleByAge() returns an array of people of a certain age", () => {});
+test("getPeopleByAge() returns an array of people of a certain age", () => {
+  const result = getPeopleByAge(people, 30);
```

Comment:
Not sure if this test is written correctly, as the returned value of person aged 30 should be undefined, as there's no person with age 30.